docs/ref/contrib/gis/install/index.txt

```diff
@@ -416,85 +415,94 @@ __ https://www.enterprisedb.com
 PostGIS
 ~~~~~~~
 
-From within the Application Stack Builder (to run outside of the installer,
-:menuselection:`Start --> Programs --> PostgreSQL 9.x`), select
-:menuselection:`PostgreSQL Database Server 9.x on port 5432` from the drop down
-menu.  Next, expand the :menuselection:`Categories --> Spatial Extensions` menu
-tree and select :menuselection:`PostGIS X.Y for PostgreSQL 9.x`.
-
-After clicking next, you will be prompted to select your mirror, PostGIS
-will be downloaded, and the PostGIS installer will begin.  Select only the
-default options during install (e.g., do not uncheck the option to create a
-default PostGIS database).
-
-.. note::
-
-    You will be prompted to enter your ``postgres`` database superuser
-    password in the 'Database Connection Information' dialog.
-
-psycopg2
-~~~~~~~~
-
-The ``psycopg2`` Python module provides the interface between Python and the
-PostgreSQL database.  Download the latest `Windows installer`__ for your version
-of Python and PostgreSQL and run using the default settings. [#]_
-
-__ http://www.stickpeople.com/projects/python/win-psycopg/
+From within the Stack Builder (to run outside of the installer,
+:menuselection:`Start --> PostgreSQL 12 --> Application Stack Builder`), select
+:menuselection:`PostgreSQL 12 (x64) on port 5432` from the drop down
+menu and click next. Expand the
+:menuselection:`Categories --> Spatial Extensions` menu tree and select
+:menuselection:`PostGIS X.Y for PostgreSQL 12`.
+
+After clicking next, you will be prompted to confirm the selected package and
+"Download directory". Click next again, this will download PostGIS and you will
+be asked to click next to begin the PostGIS installer. Select the
+following options during install:
+
+- Install PostGIS. If you choose to install the demo database you will
+  need to enter your PostgreSQL password, and confirm a name for this demo
+  database.
+- "GDAL_DATA" environment variable. Select "No" to this option we will set it
+  manually later.
+- Raster drivers, default is "No". If you know you need this feature then
+  select "Yes".
+- Raster out of DB, default is "No". If you know you need this feature then
+  select "Yes".
```

Comment:
I don't see much value in these lines.

Reply:
The installer pops up these scary alert boxes. The initial response is _Errrr... no idea_. I thought they were handy.

Reply:
We want to keep defaults, so I'm not sure :shrug: We can keep them if you prefer.

Reply:
Perhaps a rephrase though for "Select the defaults..."

Reply:
I _think_ there is some value here. The previous version says "select the defaults", but when you step through the screens it's not obvious what the defaults are. 

We could shorten it to "Select the default option ("No") to the next three dialog boxes"?